src/main/java/step1/Operator.java

```diff
@@ -0,0 +1,49 @@
+package step1;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    PLUS("+", Integer::sum),
+    SUBTRACT("-", Operator::subtract),
+    MULTIPLE("*", Operator::multiple),
+    DIVIDE("/", Operator::divide)
+    ;
+
+    private String symbol;
+
+    private BiFunction<Integer, Integer, Integer> operate;
+
+    Operator(final String symbol, final BiFunction<Integer, Integer, Integer> operate) {
+        this.symbol = symbol;
+        this.operate = operate;
+    }
+
+    private static Integer subtract(Integer a, Integer b) {
+        return a - b;
+    }
+
+    private static Integer multiple(Integer a, Integer b) {
+        return a * b;
+    }
+
+    private static Integer divide(Integer a, Integer b) {
+        if (a % b != 0) {
+            throw new IllegalArgumentException("나눗셈의 경우 결과 값을 정수로 떨어지는 값으로 한정됩니다.");
+        }
+        return a / b;
+    }
+
+    public Integer calculate(Integer a, Integer b) {
+        return operate.apply(a, b);
+    }
+
+    public static Operator of(final String symbol) {
+        return Arrays.stream(values()).filter(a -> Objects.equals(a.symbol, symbol))
+                .findFirst()
+                .orElseThrow(() -> { throw new IllegalArgumentException("사칙연산 기호가 아닙니다."); });
```

Comment:
스트림 api는 적절한 개행으로 가독성을 높여 보시면 좋을 것 같아요

> 규칙 5: 줄여쓰지 않는다(축약 금지).

람다 내부의 지역변수도 의미 있게 작성해주시면 좋을 것 같아요
람다식에 코드가 한 줄만 있는 경우 중괄호는 생략이 가능합니다
```suggestion
        return Arrays.stream(values())
            .filter(operator -> operator.symbol.equals(symbol))
            .findFirst()
            .orElseThrow(() -> new IllegalArgumentException("사칙연산 기호가 아닙니다."));
```